supernode/daemon/mgr/gc/gc_peer.go

```diff
@@ -53,6 +54,10 @@ func (gcm *Manager) gcPeers(ctx context.Context) {
 		gcPeerCount++
 	}
 
+	if timeDuring := time.Since(startTime).Seconds(); timeDuring > 2.0 {
```

Comment:
Should we make `2.0` a const. Please not hard code that in implementation.

Reply:
Agree.

Reply:
Emm... Do we need to expose it as a configuration item?

Reply:
That's a good ides. However, I think it' not necessary. It is an internal mechanism that the users almost do not care. And it will make configuration items more complex.